components/paragraphs/ParagraphResolver.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+
+import { ParagraphUnion } from "@/lib/graphql/generated/dpl-cms/graphql"
+
+import ParagraphGoVideo from "./ParagraphGoVideo/ParagraphGoVideo"
+import { ParagraphErrorBoundary } from "./paragraphErrorBoundary/paragraphErrorBoundary"
+
+function ParagraphResolver({ paragraphs }: { paragraphs: ParagraphUnion[] }) {
+  const components = {
+    ParagraphGoVideo,
+  }
+
+  return paragraphs.map((paragraph, index) => {
+    const type = paragraph?.__typename
+    if (!type) {
+      return null
+    }
+
+    const DynamicComponentType = components[type as keyof typeof components] || null
+    if (DynamicComponentType === null) return null
+
+    return (
+      <ParagraphErrorBoundary key={index}>
+        {/* @ts-ignore TODO: figure out how to type dynamically imported components */}
+        <DynamicComponentType key={index} {...paragraph} />
```

Comment:
While I think it is ok, for now, to  ignore the ts error,  I really think we should be able to handle case like this. And it is definetely possible.
If you choose to ignore this I think we should  have a jira task for it, otherwise it will will be forgotten in eternity ....eternity ... eternity _(weird echo)_